website/pages/blog/graphiql-3/index.mdx

```diff
@@ -0,0 +1,212 @@
+---
+title: GraphiQL 3 Is Here and GraphiQL 4 Is Coming
+tags: [graphql, react, graphiql]
+authors: dimitri
+date: 2023-07-01
+description: GraphiQL 3 was released, here is what was changed and what will be in GraphiQL 4.
+image: /blog-assets/graphiql-3/cover.png
+thumbnail: /blog-assets/graphiql-3/thumbnail.png
+---
+
+import { Anchor } from '@theguild/components'
+
+## Intro
+
+10 months ago
+[GraphiQL 2 was released](https://github.com/graphql/graphiql/releases/tag/graphiql%402.0.0) with a
+completely new design and new components library named
+[`@graphiql/react`](https://github.com/graphql/graphiql/tree/main/packages/graphiql-react).
+
+Unfortunately previously planned in the [roadmap](https://github.com/graphql/graphiql/issues/2328)
+most-waited features as integration [monaco-editor](https://github.com/microsoft/monaco-editor) and
+new docs explorer (aka Query Builder) were not introduced.
+
+After months and months, no progress was made, and I was called to join forces and help the
```

Comment:
I would make it a bit nicer maybe
Maybe say that you decided to join the team and push all the things on the roadmap